### ISSUE TITLE
Unable to connect to HANA via SSL

### ISSUE THREAD

**Issue by PhilipMugglestone** [SAP/sqlalchemy-hana#50]
Hi,

I'm unable to connect to a HANA system which enforces SSL. I'm using encrypt as a parameter but doesn't seem to have any effect:

```
from sqlalchemy import create_engine
engine = create_engine('hana://user:pwd@host:portnr/tenant?encrypt=true');
connection = engine.connect()
```

**DBAPIError: (hdbcli.dbapi.Error) (4321, 'only secure connections are allowed') (Background on this error at: http://sqlalche.me/e/dbapi)**

Whereas I'm able to connect OK via the Python API using the encrypt option:

```
conn = dbapi.connect(
    address="host",
    port=portnr,
    encrypt="true",
    user="user",
    password="pwd"
)
```

Thanks,
Philip

**Comment by i-pip** [comment]
Hi Phillip, did you ever find a solution to this? Encountering the same issue when pushing a python buildpack to SAP Cloud Platform via cf push

**Comment by PhilipMugglestone** [comment]
> @PhilipMugglestone Hi Phillip, did you ever find a solution to this? Encountering the same issue when pushing a python buildpack to SAP Cloud Platform via cf push

@i-pip
Hi Phillip, 
No I didn't as yet - so still using dbapi.connect. 
You can see a tutorial video here: https://www.youtube.com/watch?v=lnhlegJDT2Q&index=18&list=PLkzo92owKnVyh290bEmKED11Gcdkyoi7w 
Philip

**Comment by i-pip** [comment]
Thank you, this way works! Also found that generating the certificate manually on my local machine works. 

Procedure:

`sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout mycert.pem -out mycert.pem`

then adding the following to dbapi.connect's security parameters:

```
sslCryptoProvider = ‘openssl’,
sslKeyStore = ‘mycert.pem’,
sslTrustStore = ‘mycert.pem’,
sslValidateCertificate = ‘false’
```

as described in the comments here: https://blogs.sap.com/2018/06/18/setting-up-app-router-for-hana-xsa-with-python 

Thanks again

**Comment by jarus** [comment]
With the recent merged pull request (#54) this issue should be solved.
